code/go/0chain.net/smartcontract/storagesc/sc.go

```diff
@@ -27,11 +27,16 @@ type StorageSmartContract struct {
 }
 
 func NewStorageSmartContract() (sci.SmartContractInterface, *sci.SmartContract) {
+	var sscCopy = newStorageSmartContract()
+	return sscCopy, sscCopy.SmartContract
+}
+
+func newStorageSmartContract() *StorageSmartContract {
```

Comment:
sorry, what's that?
what was the reason to add a function with such "no-no" name, that is not used?
if it's needed, better to parametrize the original one.

ps. would be nice to rename "sscCopy" var to "scc". to reduce gradually "poor naming" cases.

Reply:
Removed dependency on sc.go